config/app.php

```diff
@@ -42,7 +42,7 @@
     |
      */
 
-    'env' => 'development',
+    'env' => env('APP_ENV', 'production'),
```

Comment:
ℹ️ Production configuration by default so I don't need to add a `.env` file to the `builds` directory